REVIEWING.md

```diff
@@ -0,0 +1,119 @@
+Breaking changes can include renaming an option without retaining the original name as an alias, removing an option entirely, or changing how an option works in a way that does not ensure backwards compatibility.
+
+Periods when it is acceptable to merge breaking changes will be widely announced.
+If it is not one of those periods, reviewers should be on the lookout for breaking changes.
+PRs with breaking changes should not be merged.
+Please guide the contributor in how to make the change in a non-breaking fashion.
+If this is not possible, deferring the PR to the next breaking change window or closing it entirely is appropriate.
+
+### Commit Messages and Hygiene
+
+Good commit messages are essential for understanding why a change was made in the future.
+Reviewers should check each commit of a PR to ensure that it has an appropriate commit message which fully and accurately explains the change and why it is being made.
+This should remain true after changes to the PR due to code review, so please re-review commit messages before merging code, to ensure they are still accurate.
+Pull requests fixing a specific issue must include a `Fixes: #xxxxx` line in the commit message.
+Full details on the `Fixes` line can be found in the [Contributor's Guide](./CONTRIBUTING.md).
+Reviewers are responsible for enforcing the guidelines in that document.
+
+Each commit in a PR should be self-contained and have a clear and distinct purpose.
+If this is not true, please encourage the contributor to squash their commits using `git rebase -i` until it is true.
+
+### Disagreements between reviewers
+
+We do not expect all reviewers to be of the same opinion during code review.
+If you see another reviewer requesting changes to a PR you do not agree with, it is perfectly acceptable to comment to that effect.
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful and are done with the goal of resolving the dispute.
+
+If a decision cannot be reached, the issue may be put to a vote, in which all Maintainers of the repository in question and all Core Maintainers can vote.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+For all branches except the main branch, this should always remain static.
```

Comment:
"should" is probably a good word here.  We may end up bumping the go version in the go.mod files of some of the RHEL branches to resolve the recent crypto issue.

Reply:
IMO RHEL policy should have nothing to do with this. I think we are all in agreement to move the RHEL branches out of this repo so these must be upstream policies only.

Reply:
Still, I think I should weaken this to say "almost always". We occasionally do need to bump for CVEs and such.

Reply:
FWIW, fedora reviews have [must and should](https://docs.fedoraproject.org/en-US/packaging-guidelines/ReviewGuidelines/#_things_to_check_on_review) . So, I'd say "should" is already kinda weak. But I'm cool either way. We could also account for special cases here: `except where the situation demands, for e.g. CVE fixes`.